modules/apps/segments/segments-service/src/main/java/com/liferay/segments/service/impl/SegmentsExperienceLocalServiceImpl.java

```diff
@@ -62,10 +63,18 @@ public SegmentsExperience addSegmentsExperience(
 			ServiceContext serviceContext)
 		throws PortalException {
 
+		int lowestPriority = _getLowestPriority(
```

Comment:
In the story, only the experiences suing the segments Anyone must be below the default experience. Is it correct to do it for all experiences?

Reply:
Yesterday, talking with Julia, we decided this is the correct behaviour, that avoid a lot of confusion.

Reply:
ohh, I didn't now that